docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -116,7 +125,7 @@ If a command line tool is used on a project file, rather than a solution file, t
 
 Specifically, when a config file is not specified explicitly on the command line, NuGet loads settings from the different config files in the following order:
 
-1. The [`NuGetDefaults.Config` file](#nuget-defaults-file), which contains settings related only to package sources.
+1. (*Uncommon*) The [`NuGetDefaults.Config` file](#nuget-defaults-file), which contains settings related only to package sources.
 1. The computer-level file.
 1. The user-level file.
 1. Files found in every folder in the path from the drive root to the current folder (where `nuget.exe` is invoked or the folder containing the Visual Studio solution). For example, if a command is invoked in `c:\A\B\C`, NuGet looks for and loads config files in `c:\`, then `c:\A`, then `c:\A\B`, and finally `c:\A\B\C`.
```

Comment:
I believe that user-level `config\*.config` is loaded after the user-level file, but before the solution directory and parents.

Reply:
I didn't quite follow this comment (user-level is loaded after user-level?), but regardless as I mentioned in other places, I'm considering enhancements like this out of scope.